libandroid-navigation/src/main/java/com/mapbox/services/android/navigation/v5/location/LocationValidator.java

```diff
@@ -3,43 +3,17 @@
 import android.location.Location;
 import android.support.annotation.NonNull;
 
-import com.mapbox.geojson.Point;
-import com.mapbox.turf.TurfConstants;
-import com.mapbox.turf.TurfMeasurement;
-
 public class LocationValidator {
 
-  private static final int ONE_SECOND_IN_MILLIS = 1000;
-
   private Location lastValidLocation;
-  private int locationAcceptableAccuracyInMetersThreshold;
-  private int locationAccuracyPercentThreshold;
-  private int locationUpdateTimeInMillisThreshold;
-  private int locationVelocityInMetersPerSecondThreshold;
+  private final int accuracyThreshold;
 
-  public LocationValidator(int locationAcceptableAccuracyInMetersThreshold, int locationAccuracyPercentThreshold,
-                           int locationUpdateTimeInMillisThreshold, int locationVelocityInMetersPerSecondThreshold) {
-    this.locationAcceptableAccuracyInMetersThreshold = locationAcceptableAccuracyInMetersThreshold;
-    this.locationAccuracyPercentThreshold = locationAccuracyPercentThreshold;
-    this.locationUpdateTimeInMillisThreshold = locationUpdateTimeInMillisThreshold;
-    this.locationVelocityInMetersPerSecondThreshold = locationVelocityInMetersPerSecondThreshold;
+  public LocationValidator(int accuracyThreshold) {
```

Comment:
Are we ok here breaking SemVer?
If so, we should document this in the release notes.